tasks/main.yml

```diff
@@ -22,7 +22,7 @@
   mount:
     name: "{{ lvm_lvmount }}"
     src: /dev/{{ lvm_vgname }}/{{ lvm_lvname }}
-    dump: "1"
-    passno: "2"
+    dump: "{{ lvm_lvmount_dump | default(1) }}"
```

Comment:
Any reason to use inline `default` in the task vs defining default values in `defaults/main.yml`?

Reply:
I'm not sure on style. I'd have been copying what I'd seen further up the same file: https://github.com/openmicroscopy/ansible-role-lvm-partition/blob/master/tasks/main.yml#L10

Reply:
Should I place in `defaults/main.yml` ?

Reply:
@sbesson @manics  - should I also remove the inline defaults in the rest of that playbook?

Reply:
From my perspective, using `defaults/main.yml` makes sense as it means one place to looks for all the default values, less places to update when making a change and you can point at the file from the top-level `README.md` but there might be downsides.

Reply:
Let's see what @manics says - I was modifying his work in the first place.

Reply:
Either is fine, depends whether you think it's worth creating another file or not. Just don't mix both styles in the same role.

Reply:
There's already a defaults file here, for a var with and empty default by the look of it. cf https://github.com/openmicroscopy/ansible-role-lvm-partition/blob/master/defaults/main.yml

Reply:
Looks like I placed that empty default in, and you created the file as boilerplate. /shrug. Adding then.